### PR TITLE
Allow the app to boot if S3 is not configured

### PR DESCRIPTION
**Pull request by jagthedrummer**
Fixes https://github.com/bullet-train-co/bullet_train/issues/1284

There may be better ways to handle this, but right now anyone who updates to a new version (or who starts a new project) is going to have trouble even booting their app in a production environment if they haven't configured S3.

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -111,6 +111,9 @@
 
   if (ENV["AWS_ACCESS_KEY_ID"] || ENV["BUCKETEER_AWS_ACCESS_KEY_ID"]).present?
     config.active_storage.service = :amazon
+  else
+    puts "WARNING! : We didn't find an active_storage.service configured so we're falling back to the local store, but it's A VERY BAD IDEA to rely on it in product unless you know what your'e doing."
```
Comment (zackgilbert) [inline config/environments/production.rb:115]: Nit: think you mean: "production"?

Comment (jagthedrummer) [reply config/environments/production.rb]: I did, indeed! Thanks for the sharp eye.

**Review by zackgilbert — APPROVED** [review]
This is great. Thank you! 

Like I mentioned in Discord, a nice additional would be a link to a BT specific doc helping people easily set this up (or directing them to a service that'll do this for them).

**Comment by jagthedrummer** [comment]
Created a separate issue for making some docs that we can link to. Gonna go ahead and merge this just to try to minimize the number of people who stub their toe on this.

**Pull request merged**